python/ray/tune/commands.py

```diff
@@ -283,3 +284,16 @@ def add_note(path, filename="note.txt"):
         print("Note updated at:", filepath)
     else:
         print("Note created at:", filepath)
+
+
+def kill(tune_address, port_forward, trial_id):
+    """Terminates trial within running TuneServer with specific trial_id.
+
+    Args:
+        tune_address (str): Address of running TuneServer.
+        port_forward (int): Port number of running TuneServer.
+        trial_id (str): ID of specific trial.
+    """
+    manager = TuneClient(tune_address=tune_address, port_forward=port_forward)
+    manager.stop_trial(trial_id)
```

Comment:
Could this be generalized to work with regexes? So you would do something like:
```python
trials = manager.get_trials()
trials_to_stop = [
    trial for trial in trials
    if re.match(trial.id, trial_id)  # This syntax might be off
]
manager.stop_trials(trials_to_stop)  # Not sure if stop_trials exist, but should be easy to implement
```

Reply:
Can a single trial_id map to multiple trials? My understanding is that a single call to `tune kill` can terminate a single trial. Or are you suggesting we can take in multiple trial_id's as input to `tune kill`?

Reply:
Ah, you're right. I thought that the trial id here was the trial name. I think this is good for now, but eventually it would be useful to be able to kill trials with name regexp.